client/blocks/stats-navigation/index.js

```diff
@@ -97,9 +111,13 @@ class StatsNavigation extends Component {
 
 export default connect( ( state, { siteId } ) => {
 	return {
-		isGoogleMyBusinessLocationConnected: isGoogleMyBusinessLocationConnectedSelector( state, siteId ),
+		isGoogleMyBusinessLocationConnected: isGoogleMyBusinessLocationConnectedSelector(
+			state,
+			siteId
+		),
 		isStore: isSiteStore( state, siteId ),
 		isJetpack: isJetpackSite( state, siteId ),
+		currentPlan: getCurrentPlan( state, siteId ),
```

Comment:
Since `currentPlan` is only used in https://github.com/Automattic/wp-calypso/pull/24258/files#diff-fbcab6691ff4fa13fc389dc4bd746990R61 to determine if the plan is a free wpcom one, why don't you resolve it here and pass the boolean prop already resolved? It could be something like:
```js
export default connect( ( state, { siteId } ) => {
const productSlug = get( getCurrentPlan( state, siteId ), 'productSlug' );
return {
    ...
    isWpComPaidPlan: ! isWpComFreePlan( productSlug )
```